### PR TITLE
update FSRS to v0.2.0

### PR DESCRIPTION
**Pull request by L-M-Sherlock**
@dae, I don't know how to deal with the translation. So I leave it for you.

### CODE REVIEW

File: rslib/src/scheduler/fsrs/weights.rs [hunk 1]
```diff
@@ -39,12 +39,12 @@ impl Collection {
         let mut anki_progress = self.new_progress_handler::<ComputeWeightsProgress>();
         let timing = self.timing_today()?;
         let revlogs = self.revlog_for_srs(search)?;
-        if revlogs.len() < 1000 {
+        if revlogs.len() < 400 {
```
Comment (user1823) [inline rslib/src/scheduler/fsrs/weights.rs:42]: Here, 400 is the number of distinct cards and not the number of revlog entries. Right?

Comment (L-M-Sherlock) [reply rslib/src/scheduler/fsrs/weights.rs:42]: It's the number of revlog entries.

Comment (user1823) [reply rslib/src/scheduler/fsrs/weights.rs:42]: What if the user has just 100 cards with 4 entries each? Assuming two button usage and 90% retention, the number of first ratings in Again and Good will be 10 and 90. I don't think that they are sufficient to generate reasonable S0 values.

If we assume that all the buttons are used, the situation is even worse.

Comment (L-M-Sherlock) [reply rslib/src/scheduler/fsrs/weights.rs:42]: FSRS has L1 regularization for initial stability fitting. When the sample size is small, FSRS doesn't allow the initial stability deviate from the default values too much.

**Comment by dae** [comment]
Thanks Jarrett! Please see my pushed commit for future reference.

**Pull request merged**